### PR TITLE
ref(ui): Remove error robot from projects page

### PR DESCRIPTION
**Pull request by maheskett**
Remove the error robot from Projects page if there's one project, and show the project instead. Continue to show the Resources section below if the user hasn’t sent a first event.

Screen Shot 2019-08-06 at 11 57 47 AM

### CODE REVIEW

File: src/sentry/static/sentry/app/components/resourceCard.jsx [hunk 1]
```diff
@@ -52,3 +43,17 @@ const ResourceCardWrapper = styled(Panel)`
...
+const StyledImg = styled('img')`
+  display: block;
+  margin: 0 auto;
+  padding-bottom: 20px;
+  height: 180px;
+`;
+
+const StyledTitle = styled('div')`
+  color: #493e54;
+  font-size: 16px;
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/resourceCard.jsx]: `theme.fontSizeLarge` is this value.

File: src/sentry/static/sentry/app/components/resourceCard.jsx [hunk 2]
```diff
@@ -52,3 +43,17 @@ const ResourceCardWrapper = styled(Panel)`
...
+
+const StyledImg = styled('img')`
+  display: block;
+  margin: 0 auto;
+  padding-bottom: 20px;
+  height: 180px;
+`;
+
+const StyledTitle = styled('div')`
+  color: #493e54;
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/resourceCard.jsx]: Is there a theme colour we can use here?

File: src/sentry/static/sentry/app/views/projectsDashboard/resources.jsx [hunk 3]
```diff
@@ -3,57 +3,37 @@ import {Flex} from 'grid-emotion';
...
+              link="https://blog.sentry.io/2018/03/06/the-sentry-workflow"
+              imgUrl={releasesImg}
+              title="The Sentry Workflow"
+            />
+            <ResourceCard
+              link="https://sentry.io/vs/logging/"
+              imgUrl={breadcrumbsImg}
+              title="Sentry vs Logging"
+            />
+            <ResourceCard link="https://docs.sentry.io/" imgUrl={docsImg} title="Docs" />
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/projectsDashboard/resources.jsx]: Does 'Docs' need to be translated?

File: src/sentry/static/sentry/app/views/projectsDashboard/resources.jsx [hunk 4]
```diff
@@ -64,10 +44,13 @@ const ResourcesWrapper = styled('div')`
...
+  padding: 30px 30px 10px 30px;
 `;
 
-const ResourcesSection = styled('div')`
-  padding: 0 30px 20px 30px;
+const ResourceCards = styled(Flex)`
+  display: grid;
+  grid-template-columns: repeat(3, auto);
+  margin-top: 25px;
+  grid-gap: 30px;
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/projectsDashboard/resources.jsx]: Should any of these use `space()` to align with the spacing grid in the app?

**Review by markstory — COMMENTED** [review]
Looks good to me. But I'll leave approval up to someone in @getsentry/growth

File: src/sentry/static/sentry/app/views/projectsDashboard/resources.jsx [hunk 5]
```diff
@@ -3,71 +3,54 @@ import {Flex} from 'grid-emotion';
...
 
-const RobotPanel = styled(Panel)`
-  margin: 30px 30px 20px 30px;
-`;
-
-const ResourcesSection = styled('div')`
-  padding: 0 30px 20px 30px;
+const ResourceCards = styled(Flex)`
+  display: grid;
+  grid-template-columns: repeat(3, auto);
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/views/projectsDashboard/resources.jsx]: 😍

**Review by evanpurkhiser — APPROVED** [review]
dis is gud

**Pull request merged**